user-guide/Reference/DataMiner_Tools/Data_Aggregator_Migrators.md

```diff
@@ -1,17 +1,59 @@
 ---
-uid: Data_Aggregator_Migrator
+uid: Data_Aggregator_Migrators
 ---
 
-# Data Aggregator Migrator
+# Data Aggregator Migrators
 
-The Data Aggregator Migrator should be used **after** upgrading the Data Aggregator DxM from version **2.x.x to 3.x.x** or higher. This **one-time migration process** is only necessary if the prior installation included [configured GQI queries](xref:Data_Aggregator_queries).
+Following migrators are available to make it easier to update existing setups to newer versions:
 
-> You can download the tool from [DataMiner Dojo](https://community.dataminer.services/download/data-aggregator-migrator/).
+* [Enabling GQI DxM](#enabling-gqi-dxm)
+* [Upgrading from version 2.x.x](#upgrading-from-version-2xx)
 
 > [!TIP]
 > See also: [Data Aggregator](xref:Data_Aggregator_DxM)
 
-## Using the Data Aggregator Migrator
+## Enabling GQI DxM
+
+The Data Aggregator MigratorToGQIDxM tool should be used to migrate any existing queries into the format supported by the GQI DxM.
+
+1. Make sure Data Aggregator version 3.1.0 or higher is installed.
+
+1. Enable the [`UseGQIDxM`](xref:Data_Aggregator_settings#gqi-dxm) setting.
+
+1. Restart the `DataMiner DataAggregator` service.
```

Comment:
Should the service not be restarted **after** the migrator has run to make sure that the correct content is loaded?

Reply:
No, everything in-memory should be updated since it's using the API calls